logzio-sender/src/test/java/io/logz/sender/LogzioSenderTest.java

```diff
@@ -172,7 +195,7 @@ public void testLoggerCantWriteToEmptyDirectory() throws Exception {
         String message1 = "Just sending something - " + random(5);
         try {
             LogzioSender testSender = getTestLogzioSender(token, type, drainTimeout, 98,
-                    tempDirectory, 10 * 1000, 10 * 1000, mockListener.getPort());
+                    tempDirectory, 10 * 1000, 10 * 1000, mockListener.getPort(), false);
```

Comment:
Just have the the same method call you new method with false, and avoid adding false everywhere